integration-tests/ledger-signer-failing-tests.spec.ts

```diff
@@ -55,9 +55,9 @@ CONFIGS().forEach(({ lib, setup }) => {
                 done();
             });
 
-            describe('Should be abble to used Ledger with wallet API', () => {
+            describe('Test that Ledger can be used with wallet API', () => {
                 jest.setTimeout(60000)
-                it('Should throw error when user declines transaction with Ledger', async (done) => {
+                it('SAs a User I want to verify that Ledger will throw an error when I declined to originate contract with Ledger\n', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"

Reply:
changed to
'Verify that Ledger will throw an error when contract origination is declined'